mula/scheduler/server/handlers/queues.py

```diff
@@ -96,9 +96,9 @@ def pop(self, queue_id: str, filters: storage.filters.FilterRequest | None = Non
                 detail="could not pop item from queue, check your filters",
             )
 
-        return models.PrioritizedItem(**p_item.model_dump())
+        return models.Task(**p_item.model_dump())
 
-    def push(self, queue_id: str, item: serializers.PrioritizedItem) -> Any:
+    def push(self, queue_id: str, item_in: serializers.Task) -> Any:
```

Comment:
Was a bit torn on the naming. Since prioritized item is not there anymore there is still the notion of when a task is put on the queue it is considered an item within the `PriorityQueue` object. Meaning renaming every instance of item to task I'd considered to create a lot of noise in the PR. I agree that it would be more descriptive and less confusing to rename all instances of item to task. So I propose to pick up renaming these in a separate PR.